R/test.R

```diff
@@ -47,6 +47,7 @@ use_test <- function(name = NULL, open = interactive()) {
   if (is.null(name)) {
     name <- get_active_r_file(path = "R")
   } else {
+    check_file_name_length(name)
```

Comment:
This should be edited to look more like the comparable locus in `use_r()`. It looks like we could be using `%||%` here too, actually.

Reply:
Good idea! Fixed in 147a626